src/components/EmptyMessage.js

```diff
@@ -0,0 +1,16 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import { RegularText } from '.';
+
+export const EmptyMessage = ({ title }) => {
+  return (
+    <RegularText small center>
```

Comment:
could setting the prop `placeholder` as well makes sense here to reduce strength of the text?

Reply:
Sure, I can try. Should I leave the size as `small`?

Reply:
good question, ähm, yes, `small` makes quite sense to me.